tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 envlist =
 ;    lint,
-    py36-django{30,31},
-    py37-django{30,31},
-    py38-django{30,31},
+    py36-django{31}
```

Comment:
Should we re-add testing django master + lint checking?

Reply:
Async is so unstable in django master that I don't think it's a good bet that channels will play nicely with it -- I checked their CI and it doesn't look like they're testing against it.

Reply:
Will add back `lint`, though.

Reply:
`lint` erroring for no reason. Will write a ticket to look into it.